public/javascripts/scripts.js

```diff
@@ -0,0 +1,15 @@
+window.addEventListener("load", function(event) {
```

Comment:
Please remove this scripts.js since it's `direct-debit-frontend` specific stuff for accessibility and not related to timeout dialogs.